src/Lib/Components/Path.vala

```diff
@@ -87,22 +83,21 @@ public class Akira.Lib.Components.Path : Component, Copyable<Path> {
         double min_y = double.MAX;
```

Comment:
It seems to me calculate_extents should not be part of the Path component, but part of the PathSegment